test/integration/test_job_environments.py

```diff
@@ -27,14 +27,30 @@
 ])
 
 
-class RunsEnvironmentJobs:
+class BaseJobEnvironmentIntegrationTestCase(integration_util.IntegrationTestCase):
+
+    framework_tool_and_types = True
+
+    @classmethod
+    def setUpClass(cls):
+        os.environ["SOME_ENV_VAR"] = "Value from env"
+        super().setUpClass()
+
+    def setUp(self):
+        super().setUp()
+        self.dataset_populator = DatasetPopulator(self.galaxy_interactor)
 
     def _run_and_get_environment_properties(self, tool_id="job_environment_default"):
         with self.dataset_populator.test_history() as history_id:
             self.dataset_populator.run_tool(tool_id, {}, history_id)
             self.dataset_populator.wait_for_history(history_id, assert_ok=True)
             self._check_completed_history(history_id)
-            return self._environment_properties(history_id)
+            job_env_properties = self._environment_properties(history_id)
+            # TODO: Remove the `if` after the Pulsar PR is merged
```

Comment:
Fine by me to wait for the new `pulsar-galaxy-lib` release. I am still not very familiar with pulsar's code base, so no rush to add me to its PyPI maintainers.

Reply:
@mvdbeek you were on the list I think, at least PyPI says you were. I did add @nsoranzo now though.

I'll do a Pulsar release right now - it is a little different than the other projects since it releases two projects at once.

Reply:
Thanks @jmchilton !

Reply:
Yes, thank you John! I bugged Nate about getting added to PyPI.